conf/flight_plans/tudelft/nederdrone_cyberzoo.xml

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="1.0" ground_alt="0" lat0="51.990634" lon0="4.376789" max_dist_from_home="60" name="Nederdrone Cyberzoo" security_height="0.4">
+  <header>
+    #include "subsystems/datalink/datalink.h"
+    #include "subsystems/electrical.h"
+    #include "subsystems/radio_control.h"
+    #include "subsystems/ahrs.h"
+    #include "firmwares/rotorcraft/guidance/guidance_indi_hybrid.h"
+    #include "firmwares/rotorcraft/navigation.h"
+    #include "subsystems/actuators/motor_mixing.h"
+</header>
+  <waypoints>
+    <waypoint lat="51.990631" lon="4.376808" name="HOME"/>
+    <waypoint name="CLIMB" x="-0.7" y="-0.6"/>
+    <waypoint alt="2.5" name="STDBY" x="2.0" y="1.1"/>
+    <waypoint name="ROPE" x="-2.4" y="1.3"/>
+    <waypoint name="TD" x="0.6" y="-1.3"/>
+    <waypoint lat="51.9905834" lon="4.3767710" name="_CZ1"/>
+    <waypoint lat="51.9906440" lon="4.3767060" name="_CZ2"/>
+    <waypoint lat="51.9906860" lon="4.3768080" name="_CZ3"/>
+    <waypoint lat="51.9906238" lon="4.3768729" name="_CZ4"/>
+  </waypoints>
+  <sectors>
+    <sector color="blue" name="CyberZoo">
+      <corner name="_CZ1"/>
+      <corner name="_CZ2"/>
+      <corner name="_CZ3"/>
+      <corner name="_CZ4"/>
+    </sector>
+  </sectors>
+  <variables>
+    <!-- <variable init="0.0f" max="1.57" min="-1.57" step="0.01" type="float" var="rope_heading"/> -->
+    <variable init="0.0f" max="1.57" min="-1.57" step="0.01" type="float" var="takeoff_pitch"/>
+    <variable init="0.15f" max="10.00" min="0.00" step="0.01" type="float" var="hook_length"/>
+  </variables>
+  <blocks>
+    <block name="Wait GPS">
+      <call_once fun="NavKillThrottle()"/>
+      <while cond="!GpsFixValid()"/>
+    </block>
+    <block name="Geo init">
+      <while cond="LessThan(NavBlockTime(), 2)"/>
+      <call_once fun="NavSetAltitudeReferenceHere()"/>
+    </block>
+    <block name="Holding point">
+      <call_once fun="NavKillThrottle()"/>
+      <set value="0.0" var="approaching_rope"/>
+      <set value="0" var="take_heading_control"/>
+      <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
+    </block>
+    <block key="r" name="Start Engine">
+      <call_once fun="NavResurrect()"/>
+      <call_once fun="NavSetWaypointHere(WP_ROPE)"/>
```

Comment:
This only sets the XY position

Reply:
I will now push a version in which also the rope altitude is set automatically